### PR TITLE
Tune comment parser for asterisks

### PR DESCRIPTION
**Pull request by tengqm**
This is a followup for #269

### CODE REVIEW

File: genref/output/md/kube-scheduler-config.v1beta2.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ The default strategy is LeastAllocated with an equal &quot;cpu&quot; and &quot;m
 <td>
    <p>DefaultConstraints defines topology spread constraints to be applied to
 Pods that don't define any in <code>pod.spec.topologySpreadConstraints</code>.
-<code>.defaultConstraints[&lowast;].labelSelectors</code> must be empty, as they are
+<code>.defaultConstraints[&amp;lowast;].labelSelectors</code> must be empty, as they are
```
Comment (kbhawkey) [inline genref/output/md/kube-scheduler-config.v1beta2.md:325]: nit: Is this correct?

**Comment by kbhawkey** [comment]
One question, other changes look good.
/lgtm
/hold

File: genref/output/md/kube-scheduler-config.v1beta2.md (hunk 1 above)
Comment (tengqm) [reply genref/output/md/kube-scheduler-config.v1beta2.md]: No. This is unexpected. Sigh...

I was trying to substitute '*' by '&lowast;', however, the goldmark package converted that '&' to '&amp;'.

After another round of tuning, I'm now inclined to give up this substitution.

**Comment by kbhawkey** [comment]
Thanks @tengqm .
/lgtm

**Pull request merged**